Docker.DotNet/Endpoints/ContainerOperations.cs

```diff
@@ -387,25 +387,26 @@ public Task StartContainerExecAsync(string id, CancellationToken cancellationTok
 
         public async Task<MultiplexedStream> StartAndAttachContainerExecAsync(string id, bool tty, CancellationToken cancellationToken)
         {
+            return await StartWithConfigContainerExecAsync(id, new ExecConfig(){Tty = tty}, cancellationToken);
```

Comment:
Since this method is async, don't I need to return MultiplexedStream not Task<MultiplexedStream>, otherwise I get this error:
<code>
C:\Coding\src\github.com\microsoft\Docker-PowerShell\src\Docker.DotNet\Docker.DotNet\Endpoints\ContainerOperations.cs(390,20): error CS4016: Since this is an async method, the return expression must be of type 'MultiplexedStream' rather than 'Task<MultiplexedStream>'
C:\Coding\src\github.com\microsoft\Docker-PowerShell\src\Docker.DotNet\Docker.DotNet\Endpoints\ContainerOperations.cs(388,46): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
</code>